list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: SQL queries and HTML are mixed, making it hard to maintain and test.
Recommendations: Separate logic from presentation by using a templating engine or at least separate PHP logic into a different file or function.